postfixadmin: 3.3.13 -> 3.3.15

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/postfixadmin/postfixadmin/releases.


meta.description for postfixadmin is: Web based virtual user administration interface for Postfix mail servers

meta.homepage for postfixadmin is: https://postfixadmin.sourceforge.io/




###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/postfixadmin/postfixadmin/releases/tag/postfixadmin-3.3.15)

- [Compare changes on GitHub](https://github.com/postfixadmin/postfixadmin/compare/postfixadmin-3.3.13...postfixadmin-3.3.15)

###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 3.3.15 with grep in /nix/store/fivrcly30nj76drzn23aaw0h4gf3w2ll-postfixadmin-3.3.15
- found 3.3.15 in filename of file in /nix/store/fivrcly30nj76drzn23aaw0h4gf3w2ll-postfixadmin-3.3.15

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2 total rebuild path(s)

2 package rebuild(s)

First fifty rebuilds by attrpath

postfixadmin
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/fivrcly30nj76drzn23aaw0h4gf3w2ll-postfixadmin-3.3.15 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A postfixadmin https://github.com/r-ryantm/nixpkgs/archive/0f35384d475321c97b2b2220ebc97c2c5bf40708.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/0f35384d475321c97b2b2220ebc97c2c5bf40708#postfixadmin
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/fivrcly30nj76drzn23aaw0h4gf3w2ll-postfixadmin-3.3.15
ls -la /nix/store/fivrcly30nj76drzn23aaw0h4gf3w2ll-postfixadmin-3.3.15/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>postfixadmin</li>
  </ul>
</details>

---

###### Maintainer pings

cc @globin for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda commented on pkgs/by-name/po/postfixadmin/package.nix:
```diff
@@ -2,13 +2,13 @@
+  version = "3.3.15";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "${pname}-${version}";
```
```suggestion
    owner = "postfixadmin";
    repo = "postfixadmin";
    rev = "postfixadmin-${version}";
```
#277994

dotlambda commented on pkgs/by-name/po/postfixadmin/package.nix:
```diff
@@ -2,13 +2,13 @@
     repo = pname;
     rev = "${pname}-${version}";
-    sha256 = "sha256-46bc34goAcRvaiyW7z0AvIcd8n61TL6vgLQ+y7nNKBQ=";
+    sha256 = "sha256-dKdJS9WQ/pPYITP53/Aynls8ZgVF7tAqL9gQEw+u8TM=";
```
```suggestion
    hash = "sha256-dKdJS9WQ/pPYITP53/Aynls8ZgVF7tAqL9gQEw+u8TM=";
```

dotlambda commented on pkgs/by-name/po/postfixadmin/package.nix:
```diff
@@ -2,13 +2,13 @@
+    sha256 = "sha256-dKdJS9WQ/pPYITP53/Aynls8ZgVF7tAqL9gQEw+u8TM=";
   };
 
   installPhase = ''
```
```suggestion
  installPhase = ''
    runHook preInstall

```

dotlambda committed:
postfixadmin: 3.3.13 -> 3.3.15

Diff: https://github.com/postfixadmin/postfixadmin/compare/postfixadmin-3.3.13...postfixadmin-3.3.15

Changelog: https://github.com/postfixadmin/postfixadmin/releases/tag/postfixadmin-3.3.14
           https://github.com/postfixadmin/postfixadmin/releases/tag/postfixadmin-3.3.15

dotlambda commented:
> Update bundled smarty (4.5.4); PostfixAdmin should not be vulnerable to https://github.com/advisories/GHSA-4rmg-292m-wg3w mentioned in the release etc.

Should this be backported nevertheless?
